### PR TITLE
Prepare 1.1.0 release.

### PR DESCRIPTION
**Pull request by Natim**
* [x] Version dependencies
* [x] Update changelog
* [x] Update version in docs/conf.py
* [ ] Tag vX.Y.Z
* [ ] Close milestone in Github
* [ ] Publish on Pypi
* [ ] Add entry in Github release page
* [ ] Create next milestone in Github
* [ ] Send mail to ML
* [ ] Blog about it

### CODE REVIEW

**Comment by Natim** [comment]
https://github.com/mozilla-services/kinto/releases/tag/1.1.0

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,7 +7,22 @@ This document describes changes between each past release.
...
+
+
+**Bug fixes**
+
+- ETag not updated on collection update (#80)
+
+
+**Internal changes**
+
+- Use py.test to run tests instead of nose.
```
Comment (ametaireau) [inline CHANGELOG.rst]: We should link the pull request from @magopian here (and thank him)

File: docs/conf.py [hunk 2]
```diff
@@ -120,6 +120,7 @@
 #  dir menu entry, description, category)
 texinfo_documents = [
   ('index', 'Kinto', u'Kinto Documentation',
-   u'Mozilla Services — Da French Team', 'Kinto', 'One line description of project.',
+   u'Mozilla Services — Da French Team', 'Kinto',
+   'A storage service for data sync, share with fine-grained permissions.',
```
Comment (ametaireau) [inline docs/conf.py]: I would say "data storage, synchronization and sharing".

File: CHANGELOG.rst (hunk 1 above)
Comment (leplatrem) [inline CHANGELOG.rst]: And would also suggest to re-run ``pip install -r dev-requirements`` for those who have an existing setup

**Pull request merged**